### PR TITLE
Restored fulltext search functionality

### PR DESCRIPTION
**Pull request by tsteinruecken**
This should restore basic fulltext search capability in ViUR.
(Backported from a fixed project, should probably be tested separately before merging)

### CODE REVIEW

File: bones/stringBone.py [hunk 1]
```diff
@@ -175,11 +175,11 @@ def buildDBSort(self, name, skel, dbFilter, rawFilter):
 				dbFilter.order(order)
 		return (dbFilter)
 
-	def getSearchTags(self, valuesCache, name):
-		res = []
-		if not valuesCache.get(name):
-			return (res)
-		value = valuesCache[name]
+	def getSearchTags(self, skeltonValues, name):
```
Comment (sveneberth) [inline bones/stringBone.py:178]: @tsteinruecken There ist a typo. It must be _skel**e**tonValues_

File: bones/textBone.py [hunk 2]
```diff
@@ -285,26 +285,36 @@ def getReferencedBlobs(self, valuesCache, name):
...
 		return newFileKeys
 
-	def getSearchTags(self, valuesCache, name):
-		res = []
-		if not valuesCache.get(name):
-			return (res)
-		if self.languages:
-			for v in valuesCache.get(name).values():
-				value = HtmlSerializer(None).sanitize(v.lower())
+	def getSearchTags(self, skeltonValues, name):
```
Comment (sveneberth) [inline bones/textBone.py:288]: Here as well...

File: bones/textBone.py [hunk 3]
```diff
@@ -285,26 +285,36 @@ def getReferencedBlobs(self, valuesCache, name):
...
+		res = set()
+		value = skeltonValues[name]
```
Comment (sveneberth) [inline bones/textBone.py:290]: Here as well...

**Comment by phorward** [comment]
@sveneberth easiest will be to open a separate pull request on this.

**Pull request merged**